doc/api/debugger.md

```diff
@@ -185,9 +185,8 @@ flag instead of `--inspect`.
 
 ```console
 $ node --inspect index.js
-Debugger listening on 127.0.0.1:9229.
-To start debugging, open the following URL in Chrome:
-    chrome-devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws=127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
+Debugger listening on ws://127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
```

Comment:
Doesn't this URL no longer match the output format (e.g. missing `/bundled/`, etc.) from inspector_socket_server.cc?

Reply:
At some point, what is actually printed to the console changed. This is what is currently displayed.